app/models/meeting.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 class Meeting < ApplicationRedisRecord
-  define_attribute_methods :id, :server_id
+  define_attribute_methods :id, :server_id, :password
```

Comment:
Please call the attribute `moderator_pw` so we don't get it confused with the attendee password.